package.json

```diff
@@ -29,6 +30,8 @@
     "faker": "Marak/faker.js#master",
     "graphql-request": "^1.8.2",
     "neo4j-driver": "^1.7.5",
-    "npm-run-all": "^4.1.5"
+    "neode": "^0.2.16",
+    "npm-run-all": "^4.1.5",
+    "slug": "^1.1.0"
```

Comment:
great that you only needed to add only two packages to the root-level package.json :smile_cat: